blinky-back-end/Program.cs

```diff
@@ -1,6 +1,26 @@
 var builder = WebApplication.CreateBuilder(args);
+
+builder.Services.AddCors(options => {
+    options.AddPolicy(name: "PolicyOne", 
```

Comment:
I don't think there is a need to name your policy as you just have one.
You can use `options.AddDefaultPolicy` and `app.UseCors()` to simplify this.

https://learn.microsoft.com/en-us/aspnet/core/security/cors?view=aspnetcore-7.0#cors-with-default-policy-and-middleware